src/twisted/conch/ssh/keys.py

```diff
@@ -1006,7 +1153,8 @@ def privateBlob(self):
                     common.MP(data['q']) + common.MP(data['g']) +
                     common.MP(data['y']) + common.MP(data['x']))
         else:
```

Comment:
As above, this used to error on unknown key types, now it doesn't (or rather it errors weirdly). Can we re-establish that old behaviour?